generators/server/generator.mjs

```diff
@@ -523,7 +523,7 @@ export default class JHipsterServerGenerator extends BaseApplicationGenerator {
                 field
               )} \nHibernate JPA 2 Metamodel does not work with Bean Validation 2 for LOB fields, so LOB validation is disabled`
             );
-            field.validation = false;
+            field.fieldValidate = false;
```

Comment:
`fieldValidation` (to align with `fieldHasValidation`)?

Reply:
Probably should be ok if there is no jdl support on `fieldValidation`.
In any case `field.validation` doesn't exist.